setup.py

```diff
@@ -14,11 +14,9 @@
     ]
   },
   package_dir={'catkin_sphinx': 'src/catkin_sphinx'},
-  # The original author was Troy Straszheim
-  # Previous maintainers include: Issac Trotts
-  # The current maintainer is William Woodall
-  author="William Woodall",
-  author_email="william@osrfoundation.org",
+  author = "Troy Straszheim, Issac Trotts, William Woodall",
+  author_email = "straszheim@willowgarage.com, itrotts@willowgarage.com, william@openrobotics.org",
+  maintainer="ROS Infrastructure Team",
```

Comment:
Is there a `maintainer_email` that we should set?

Reply:
After discussing this, we decided that providing a link to the issue tracker should be sufficient. 41dde18 adds that.

Reply:
After discussing this, we decided that providing a link to the issue tracker should be sufficient. 41dde18 adds that.